src/mozilla-traffic-cop.js

```diff
@@ -26,11 +26,19 @@ if (typeof Mozilla === 'undefined') {
  *
  *
  * @param Object config: Object literal containing the following:
- *      String id (required): Unique-ish string for cookie identification.
+ *      [String] id (required): Unique-ish string for cookie identification.
  *          Only needs to be unique to other currently running tests.
- *      Number cookieExpires (optional): Number of hours browser should remember
- *          the variation chosen for the user. Defaults to 24 (hours). A value
- *          of 0 will result in a session-length cookie.
+ *      [Number] cookieExpires (optional): Number of hours browser should
+ *          remember the variation chosen for the user. Defaults to 24 (hours).
+ *          A value of 0 will result in a session-length cookie.
+ *      [Boolean] storeReferrerCookie (optional): Flag to specify whether or not
+ *          original HTTP referrer should be placed in a cookie for later use.
+ *          Defaults to true.
+ *      [Function] customCallback (optional): Arbitrary function to run when
+ *          a variation (or lack thereof) is chosen. This function will be
+ *          passed the variation value (if chosen), or the value of
+ *          noVariationCookieValue if no variation was chosen. *Specifying this
```

Comment:
An experiment can either perform a redirect or execute arbitrary JS, but not both. A redirect is really just a longer way to execute a callback, as the query param can be used to trigger loading of any additional JS (or CSS/HTML) on the resulting variation.

I set up the page 3 variation to try and mimic the current situation on `/new` - one funnelcake redirect and a separate survey link injection. They are unrelated, but target the same audience.

> Could this skew data in the real world?

In some situations, yes, certainly. I think it's up to the implementor to determine what's acceptable.